wai-logger/Network/Wai/Logger.hs

```diff
@@ -96,26 +96,17 @@ data ApacheLoggerActions = ApacheLoggerActions {
   , logRemover :: IO ()
   }
 
--- | Logger Type.
-data LogType = LogNone                     -- ^ No logging.
-             | LogStdout BufSize           -- ^ Logging to stdout.
-                                           --   'BufSize' is a buffer size
-                                           --   for each capability.
-             | LogFile FileLogSpec BufSize -- ^ Logging to a file.
-                                           --   'BufSize' is a buffer size
-                                           --   for each capability.
-                                           --   File rotation is done on-demand.
-             | LogCallback (LogStr -> IO ()) (IO ())
-
 ----------------------------------------------------------------
 
```

Comment:
Actually i'd like to refract wai-logger completely, remove Date.hs and use new fast-logger implementation, export `clockDateCacher = newTimeCacher simpleTimeFormat`, but i'm not sure if i can safely remove `ZonedDate/DateCacheGetter/DateCacheUpdater`. AFAICT no package use these exports.